module/rds_account.py

```diff
@@ -0,0 +1,656 @@
+#!/usr/bin/python
+#
+# Copyright 2017 Alibaba Group Holding Limited.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see http://www.gnu.org/licenses/
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'core',
+                    'version': '1.0'}
+
+DOCUMENTATION = """
+---
+module: rds_account
+short_description: Create account, delete account, reset instance password, reset account, grant account permission
+and revoke account permission in RDS
+description:
+     - Create account, delete account, reset instance password, reset account, grant account permission and revoke
+     account permission in RDS
+common options:
+  alicloud_access_key:
+    description:
+      - Aliyun Cloud access key. If not set then the value of the `ALICLOUD_ACCESS_KEY`, `ACS_ACCESS_KEY_ID`, 
+        `ACS_ACCESS_KEY` or `ECS_ACCESS_KEY` environment variable is used.
+    required: false
+    default: null
+    aliases: ['acs_access_key', 'ecs_access_key','access_key']
+  alicloud_secret_key:
+    description:
+      - Aliyun Cloud secret key. If not set then the value of the `ALICLOUD_SECRET_ACCESS_KEY`, `ACS_SECRET_ACCESS_KEY`,
```

Comment:
Please change 'ALICLOUD_SECRET_ACCESS_KEY' to 'ALICLOUD_SECRET_KEY' in the line 42.